thoth/package_extract/image.py

```diff
@@ -546,6 +546,47 @@ def get_image_size(path: str) -> int:
     return total_size
 
 
+def _get_cuda_version(path: str) -> dict:
+    """Get the cuda version."""
+    res = dict()
+    # gathering version from version.txt file
+    version_path = path + "/usr/local/cuda/version.txt"
+    if os.path.isfile(version_path):
+        with open(version_path, "r") as f:
+            output = f.read()
+            lines = output.split()
+            if len(lines) != 3:
+                _LOGGER.info(
+                    "Unable to detect CUDA version from usr/local/cuda/version.txt output: %r",
+                    output,
+                )
+            else:
+                res["/usr/local/cuda/version.txt"] = lines[-1]
+
+    # gathering version from nvcc command
+    nvcc_path = glob.glob(path + '/usr/local/cuda-*/bin/nvcc')
```

Comment:
Here, we try to find a path of `nvcc` binary, as I have checked in `docker.io/nvidia/cuda`, it was at this location, the only thing that could be different is `cuda-VERSION`, where `VERSION` is changing number, so I put `*` to find it.

Reply:
Nice! Just to double check, can we have multiple CUDA versions installed? If so, which one do we report? Do we produce a warning if we find more than one CUDA version?